### DISCUSSION TITLE
How to apply poster to Oscar collections?

### DISCUSSION THREAD

**Discussion by dmfreck** [meisnate12/Plex-Meta-Manager#957] [Q&A] [answered]
Hi - I'm looking for a way to have a generic "Oscars" poster that I dynamically write the years for the categories (1970-1979, etc) instead of Plex's ugly quad poster layout. Does PMM support this?

![image](https://user-images.githubusercontent.com/55361968/178042790-63900bd0-0ac1-4f9e-bfa7-f9748106791b.png)

**Comment by chazlarson** [comment, marked as answer]
There is not as yet a way within PMM to dynamically create posters.

You can, of course, assign posters to collections, and you could create a bunch of these ahead of time and then just let the asset directory pipeline pick them up.

**Reply by dmfreck** [reply]
Thank you! Yah - I'd suspected that I might need to just create these ahead of time but was trying to get out of some work.
